crates/indexer/migrations/2022-12-29-222902_curr_table_items/up.sql

```diff
@@ -0,0 +1,35 @@
+-- Your SQL goes here
+CREATE TABLE current_table_items (
+  table_handle VARCHAR(66) NOT NULL,
+  -- Hash of the key for pk since key is unbounded
```

Comment:
Could you explain this a bit more? Not sure why the table handle can't be the PK?

Reply:
Table item is part of the PK. The PK is table handle and key, but the key is unbounded so we need hash of key.